redis/redis.go

```diff
@@ -187,3 +187,27 @@ type SlowLog struct {
 	// ClientName is the name set via the CLIENT SETNAME command (4.0 only).
 	ClientName string
 }
+
+// Latency represents a redis LATENCY LATEST
```

Comment:
Always end comments with a full stop.
```suggestion
// Latency represents a redis LATENCY LATEST.
```